syntaxes/yml.tmLanguage

```diff
@@ -8,6 +8,14 @@
     <string>Yseop Markup Language</string>
```

Comment:
So there is

* this weird XML-ish version
* the generated JSON one
* the weird one with the overlong extension `YAML-tmLanguage` (gosh, my renaming scripts assume anything longer than “torrent” can’t be an extension)?

I don’t get what this version, here (XML-ish), does or where it comes from.

Reply:
I guess we can now remove completely this file.

Reply:
Try and see what happens :+1: